### PR TITLE
Redux material new

### PR DESCRIPTION
**Pull request by kimberlythegeek**

### CODE REVIEW

File: src/utils/searchViewHelper.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  updateSearchResults,
+  updateRepository,
+} from '../reducers/SearchSlice';
+import {
+  fetchRecentRevisions,
+  fetchRevisionByID,
+  fetchRevisionsByAuthor,
+} from '../thunks/searchThunk';
+
+const SearchViewHelper = {
```
Comment (beatrice-acasandrei) [inline src/utils/searchViewHelper.js:13]: Usually constants names are written using the lower camel case rule.

File: src/thunks/searchThunk.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const TREEHERDER = 'https://treeherder.mozilla.org';
```
Comment (beatrice-acasandrei) [inline src/thunks/searchThunk.js:3]:
```suggestion
const treeherderBaseURL = 'https://treeherder.mozilla.org';
```

File: src/utils/searchViewHelper.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  handleChangeSearch(event) {
+    const search = event.target.value;
+    const { repository } = store.getState().search;
+    store.dispatch(updateSearchValue(search));
+
+    // If search input is cleared, clear results
+    if (search === '') store.dispatch(updateSearchResults([]));
+
+    if (repository === '') {
+      console.log('Please select a repository');
```
Comment (beatrice-acasandrei) [inline src/utils/searchViewHelper.js:48]: `console.log` triggers a linting error.

**Review by beatrice-acasandrei — APPROVED** [review]

**Comment by kimberlythegeek** [comment]
fixes #18 #3 #1 #2

**Pull request merged**